### DISCUSSION TITLE
How do I show window controls/titlebar on my Waydroid window?

### DISCUSSION THREAD

**Discussion by alshdavid** [waydroid/waydroid#1483] [Q&A]
I just installed waydroid and configured the height/width of the device. Is there a way to enable the window title bar / window controls for the waydroid window?

![image](https://github.com/user-attachments/assets/884c4c42-3442-4443-ae80-ed6d3ef59da5)

Currently it's just a floating borderless window 

I'm on Fedora 40 and set the device dimensions [from here](https://github.com/waydroid/waydroid/issues/700). As an extra question, can I enable setting the device dimensions via window resizing?

**Comment by KnoyanMitsu** [comment]
+1 

need feature window controls/titlebar waydroid
